cmd/api/src/api/registration/v2.go

```diff
@@ -148,7 +148,10 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 		routerInst.GET("/api/v2/graphs/shortest-path", resources.GetShortestPath).Queries(params.StartNode.String(), params.StartNode.RouteMatcher(), params.EndNode.String(), params.EndNode.RouteMatcher()).RequirePermissions(permissions.GraphDBRead),
 		// TODO discuss if this should be a post endpoint
 		routerInst.GET("/api/v2/graph-search", resources.GetSearchResult).RequirePermissions(permissions.GraphDBRead),
+
+		// Cypher Queries API
 		routerInst.POST("/api/v2/graphs/cypher", resources.CypherSearch).RequirePermissions(permissions.GraphDBRead),
+		routerInst.GET("/api/v2/saved-queries", resources.ListSavedQueries).RequirePermissions(permissions.AuthManageSelf),
```

Comment:
Is the `AuthManageSelf` permission required for this endpoint?

Reply:
I'd argue that AuthManageSelf makes more sense for ListPermissions and ListRoles because those are both related to auth management. I do think we need a permission here, but maybe it would be worth having a short discussion on what that permission should be (either an existing permission or a new one)